src/components/logo/Logo.js

```diff
@@ -0,0 +1,27 @@
+import { Box } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+
+const Logo = () => {
+  const { classes } = useStyles()
+  return (
+    <Box>
```

Comment:
- Is it any reason to wrap Box into another Box?
- src of image = usually located in folderr public in root folder
- take a look plz 
https://mui.com/material-ui/icons/ 
c 
From my perspective using SvgIcon is more preferable

Reply:
There is no Image in MUI, so I used <Box> with component = "img"
When I write:
   `  <Box component="img" src="/img/logo.svg" ></Box>`
I get in html:  ` <img" src="/img/logo.svg" /> `
In fact  the second <Box>  is used to get <img>

For the SvgIcon I only found:
```
    <SvgIcon {...props}>
      <path d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z" />
    </SvgIcon>
```
I have an image in format SVG, but I don't have a path for this image. Or have I understood something wrong?